packaging/opensuse/snapd.spec

```diff
@@ -91,6 +106,9 @@ BuildRequires:  pkg-config
 BuildRequires:  python-docutils
 BuildRequires:  python3-docutils
 BuildRequires:  squashfs
+# Due to: rpm -q --whatprovides /usr/share/pkgconfig/systemd.pc
```

Comment:
Shouldn't that be `/usr/lib/pkgconfig..`?

Reply:
That's what is actually on the system and that is what pkg-config picks up. Why did you expect it to be in /usr/lib?

Reply:
I've always seen ${libdir}/pkgconfig, but I see ${datadir}/pkgconfig used too. On my system:

```
$ ls /usr/lib/pkgconfig | wc -l
1013
$ ls /usr/share/pkgconfig | wc -l
56
```
Another one with almost vanilla Fedora cloud image:
```
$ ls /usr/lib64/pkgconfig/ | wc -l
2
$ ls /usr/share/pkgconfig/ | wc -l
4
```

Looks like it's yet another thing that may vary across distros.

Reply:
I can remove the comment but the point is that we needed that .pc file.